modeltranslation_wagtail/models.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import absolute_import
+
+import django
+
+from wagtail.contrib.settings.models import BaseSetting
+from wagtail.wagtailcore.models import Page
+from wagtail.wagtailsnippets.models import get_snippet_models
+
+from .patch_wagtailadmin import WagtailTranslator
+
+
+def autodiscover():
+    """
+    Auto-discover INSTALLED_APPS translation.py modules and fail silently when
+    not present. This forces an import on them to register.
+    Also import explicit modules.
```

Comment:
This is already done by `modeltranslation` -- so when you need this, it's because you didn't put `modeltranslation` in `INSTALLED_APPS`

Reply:
yes, you are right. It looks like I took this from `wagtail_modeltranslation` before they started monkey-patching `django-modeltranslation`. Unnecessary indeed.